src/Files.App/Actions/Open/OpenContextMenuAction.cs

```diff
@@ -0,0 +1,79 @@
+﻿using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.WinUI.UI;
+using Files.App.Commands;
+using Files.App.Extensions;
+using Files.App.Filesystem;
+using Files.App.UserControls;
+using Files.App.UserControls.Widgets;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Controls.Primitives;
+using Microsoft.UI.Xaml.Input;
+using System.Threading.Tasks;
+using Windows.Foundation;
+using Windows.System;
+
+namespace Files.App.Actions
+{
+	internal class OpenContextMenuAction : IAction
+	{
+		public string Label { get; } = "OpenContextMenu".GetLocalizedResource();
+		public string Description => "TODO: Need to be described.";
+
+		public HotKey HotKey { get; } = new(VirtualKey.Enter, VirtualKeyModifiers.Control);
+
+		public bool IsExecutable
+		{
+			get
+			{
+				if (App.LastOpenedFlyout is CommandBarFlyout { IsOpen: true })
+					return false;
+
+				var element = GetFocusedElement();
+				if (element is null)
+					return false;
+
+				return element.ContextFlyout is FlyoutBase { IsOpen: false }
+					|| element.DataContext is INavigationControlItem or WidgetCardItem;
```

Comment:
```suggestion
				return element.ContextFlyout is FlyoutBase { IsOpen: false } ||
					element.DataContext is INavigationControlItem or WidgetCardItem;
```

Reply:
I don't understand this change.

Reply:
In most of the codebase we align `&&`s and `||`s on the previous line